config/config_default.cmake

```diff
@@ -61,6 +61,8 @@ set(CONFIG_TFM_LAZY_STACKING            OFF         CACHE BOOL      "Enable/disa
 
 set(CONFIG_TFM_DOORBELL_API             ON          CACHE BOOL      "Enable the doorbell APIs")
 
+set(CONFIG_TFM_LOG_SHARE_UART           OFF         CACHE BOOL      "TF-M shares the UART log instance with the application. Only use it during boot and in fault handling")
+
 ############################ Platform ##########################################
 
 set(TFM_MULTI_CORE_TOPOLOGY             OFF         CACHE BOOL      "Whether to build for a dual-cpu architecture")
```

Comment:
missing commit sauce tags.

IMHO since this is an old bug it should not be treated as a hotfix and we should do a fromlist instead of toup.

But toup is acceptable if you prefer.